crates/stc_ts_type_checker/tests/tsc.rs

```diff
@@ -755,7 +778,8 @@ fn do_test(file_name: &Path, spec: TestSpec, use_target: bool) -> Result<(), Std
 
     let extra_err_count = extra_errors.len();
     stats.required_error += expected_errors.len();
-    stats.extra_error += extra_err_count;
+    stats.extra_error += extra_err_count - unimplemented_count;
+    stats.unimplemented = unimplemented_count;
```

Comment:
I don't really know if it should be handled more structurally - with removing such errors from extra errors vector and handling it differently in debug output below. I can do it, if it's needed.